scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -13,7 +13,7 @@ const ROOT = 'root';
 const SCOPED = 'scoped';
 const ROOT_THEME = `${ROOT}-theme.scss`;
 const SCOPED_THEME = `${SCOPED}-theme.scss`;
-
+const context = '[terra-theme-aggregator]';
```

Comment:
nitpick, can we add a newline here to separate the context variable and next code block?